<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoPracownicy</class>
 <widget class="QDialog" name="OknoPracownicy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(42, 31, 48);
color: white;

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="DodajEdytujLayout">
     <item>
      <layout class="QVBoxLayout" name="DodajLayout">
       <item>
        <widget class="QLabel" name="DodajLabel">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Dodaj pracownik√≥w do bazy danych</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="DodajImieLayout">
         <item>
          <spacer name="DodajImieLewySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="DodajImieLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Imie</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="DodajImie">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="DodajImiePrawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="DodajNazwiskoLayout">
         <item>
          <spacer name="DodajNazwiskoLawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="DodajNazwiskoLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Nazwisko</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="DodajNazwisko">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="DodajNazwiskoPrawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="DodajWiekLayout">
         <item>
          <spacer name="DodajWiekLewySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="DodajWiekLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Wiek</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="DodajWiek">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="DodajWiekPrawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="DodajButtonLayout">
         <item>
          <spacer name="DodajButtonLewySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="DodajButton">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white</string>
           </property>
           <property name="text">
            <string>Dodaj</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="DodajButtonPrawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox_4124">
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(240, 240, 93);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="EdytujLayout">
       <item>
        <widget class="QLabel" name="EdytujLabel">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Edytuj pracownik√≥w w bazie danych</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="EdytujId_2">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="EdytujId">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>45</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="EdytujImieLayout">
         <item>
          <spacer name="EdytujImieLewySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="EdytujImieLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Imie</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="EdytujImie">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="EdytujImiePrawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="EdytujNazwiskoLayout">
         <item>
          <spacer name="EdytujNazwiskoLewySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="EdytujNazwiskoLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Nazwisko</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="EdytujNazwisko">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="EdytujNazwiskoPrawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="EdytujWiekLayout">
         <item>
          <spacer name="EdytujWielLewySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="EdytujWiekLabel">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Wiek</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="EdytujWiek">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="EdytujWiekPrawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="EdytujButtonLayout">
         <item>
          <spacer name="EdytujButtonLewySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="EdytujButton">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid white</string>
           </property>
           <property name="text">
            <string>Edytuj</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="EdytujButtonPrawySpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="TytulLayout">
     <item>
      <widget class="QPushButton" name="menuButton_pracownicy">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid white</string>
       </property>
       <property name="text">
        <string>Powr√≥t do menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pracownicy</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="WyswietlUsunLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="wyswietlPracownikoButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white</string>
         </property>
         <property name="text">
          <string>Wy≈õwietl Pracownik√≥w</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="WyswietlPracownikow">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>250</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: white;
color: black;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox_4">
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(240, 240, 93);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="UsunGornySpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="UsunLabel">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Usu≈Ñ rekord z bazy danych</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="UsunID">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160000</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UsunButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white</string>
         </property>
         <property name="text">
          <string>Usu≈Ñ</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="UsunDolnySpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="verticalGroupBox_476868">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(240, 240, 93);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
